src/main/java/com/google/sps/ImageTextDectector.java

```diff
@@ -14,103 +14,45 @@
 
 package com.google.sps;
 
-import com.google.cloud.vision.v1.AnnotateImageRequest;
-import com.google.cloud.vision.v1.AnnotateImageResponse;
-import com.google.cloud.vision.v1.BatchAnnotateImagesResponse;
-import com.google.cloud.vision.v1.EntityAnnotation;
-import com.google.cloud.vision.v1.Image;
-import com.google.cloud.vision.v1.ImageAnnotatorClient;
+import com.google.sps.data.ShoppingListTextEntry;
 import java.io.IOException;
-import java.util.ArrayList;
 import java.util.List;
 
 /**
- * This class generates a shopping list based on image 
- * 1) It uses cloudVisionAPI to scan an image
- * containing shopping list items and detect text from it.
- * 2) It then uses an algorithm to create shopping sentences 
- * (queries) from the text and their position.
- * 3) This list of queries is returned to the Servlet from the
- * method 'imageToShoppingListExtractor'.
+ * This class generates a shopping list based on image 1) It uses cloudVisionAPI to scan an image
+ * containing shopping list items and detect text from it. 2) It then uses an algorithm to create
+ * shopping sentences (queries) from the text and their position. 3) This list of queries is
+ * returned to the Servlet from the method 'extractShoppingList'.
```

Comment:
This change is not necessary. I'm assuming the linter did this trying to format comments.

Reply:
Yes it was done by the linter, um is it fine or should I revert it back?

Reply:
Revert it back. It's easier to read. But I'm not sure how to prevent the linter from doing it again next time.